examples/seismic/acoustic/acoustic_example.py

```diff
@@ -69,9 +67,9 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
     if constant:
         # With  a new m as Constant
         m0 = Constant(name="m", value=.25, dtype=np.float32)
-        solver.forward(save=full_run, m=m0)
+        solver.forward(save=full_run and not checkpointing, m=m0)
```

Comment:
wait, why? I don't link this combined condition here

Reply:
If you use checkpointing, you do not need to store the whole time history, so save should only be True without checkpointing.

Reply:
yes, sorry there was a TYPO in my original comment -- link --> like

what I meant was that this deserves to be hoisted to a separate line a be properly commented